src/inputs/search/query-search/PQuerySearch.vue

```diff
@@ -126,16 +128,28 @@ export default defineComponent({
     },
     setup(props, context: SetupContext) {
         const { slots, emit } = context;
+        const state = reactive({
+            visibleMenu: ref<boolean>(false),
+            value: ref(props.value),
+        });
```

Comment:
```suggestion
        const state = reactive({
            visibleMenu: false,
            value: props.value,
        });
```

`reactive` function makes an object's all properties reactive.
So `ref` function is not necessary!